src/test/scala/nl.knaw.dans.easy.deposit/servlets/IntegrationSpec.scala

```diff
@@ -206,6 +206,20 @@ class IntegrationSpec extends TestSupportFixture with ServletFixture with Scalat
   "scenario: create - ... - sumbit" should "create submitted dataset copied from a draft" in {
     val uuid: String = setupSubmittedDeposit
 
+    // resubmit fails
+    val props = testDir / s"drafts/foo/$uuid/deposit.properties"
+    props.write(props.contentAsString.replace("SUBMITTED", "DRAFT"))
+    authMocker.expectsUserFooBar
+    put(
+      uri = s"/deposit/$uuid/state",
+      headers = Seq(fooBarBasicAuthHeader),
+      body = """{"state":"SUBMITTED","stateDescription":"blabla"}"""
+    ) {
+      status shouldBe CONFLICT_409
+      body shouldBe s"The deposit (UUID $uuid) already exists in the submit area. Possibly due to a resubmit."
```

Comment:
Not sure about the wording. 

Stumbled on it when manualing resetting a status from submitted to draft without deleting the directory in ingest-flow-inbox. But might also occur when a user submits twice concurrently (in different tabs?) 
https://github.com/DANS-KNAW/easy-deposit-api/blob/4e60de9256c601cfa1526d2ae4d25e1cf8ef8263/src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala#L87-L88
https://github.com/DANS-KNAW/easy-deposit-api/blob/4e60de9256c601cfa1526d2ae4d25e1cf8ef8263/src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala#L116-L118



Is also related to https://drivenbydata.atlassian.net/browse/EASY-2027 implementation of unsubmit.